doc/tutorial/search/solution2.cpp

```diff
@@ -0,0 +1,31 @@
+#include <seqan3/io/stream/debug_stream.hpp>
+#include <seqan3/search/algorithm/search.hpp>
+
+using namespace seqan3;
+using namespace seqan3::search_cfg;
+
+int main()
+{
+	{
```

Comment:
this is not pretty :nail_care: I would just give variables unique names.

Reply:
Mhm, I'm doing this in multiple solutions.
I could add two functions (`void run_text_single()` and `void run_text_collection()`) and then just call both in the main?